### PR TITLE
add support for quantum coins

### PR DESCRIPTION
**Pull request by cyggy**
add support for quantum coins but only for the #premium 1%ers

### CODE REVIEW

**Review by dngfx — CHANGES_REQUESTED** [review]
Please wrap the or statement

File: modules/coins.py [hunk 1]
```diff
@@ -260,7 +261,7 @@ def flip(self, event):
             raise utils.EventError("%s: You don't have enough coins to bet" % event["user"].nickname)
 
         chosen_side = random.SystemRandom().choice(list(SIDES.keys()))
-        win = side_name == chosen_side
+        win = side_name == chosen_side or side_name == "heils" and event["user"].nickname == "cygnus"
```
Comment (CaptainDreamcast) [inline modules/coins.py:264]: ummmm, a bit problematic

Comment (cyggy) [reply modules/coins.py:264]: fist my ass hows that for problematic

**Comment by dngfx** [comment]
also no

File: modules/coins.py (hunk 1 above)
Comment (CaptainDreamcast) [reply modules/coins.py:264]: no I get it now, thanks for the explanation I'll close the comment

**Comment by cyggy** [comment]
CLOSED WONTFIX CANT REPRODUCE ERROR

**Pull request closed without merging**